kolibri/core/public/api_urls.py

```diff
@@ -80,4 +79,9 @@
         get_public_file_checksums,
         name="get_public_file_checksums",
     ),
+    url(
+        r"syncqueue/",
```

Comment:
This made it 
![image](https://github.com/learningequality/kolibri/assets/1008178/89b998b9-2d24-4775-99f5-fe659df4bae6)